pkgs/applications/video/shotcut/default.nix

```diff
@@ -45,14 +48,12 @@ mkDerivation rec {
     qtwebsockets
     qtquickcontrols2
     qtgraphicaleffects
+    fftw
   ];
 
   env.NIX_CFLAGS_COMPILE = "-I${mlt.dev}/include/mlt++ -I${mlt.dev}/include/mlt";
```

Comment:
this can apparently be omitted

Reply:
Omitting this causes the build to fail
```
error: builder for '/nix/store/15qln57gip1vlnakkiyg1pdpavvz1n7j-shotcut-22.12.21.drv' failed with exit code 1;
       last 10 log lines:
       > --   No package 'fftw' found
       > CMake Error at /nix/store/l341npqn3j8pwqq1nqgj5cqhnfbzhphd-cmake-3.25.1/share/cmake-3.25/Modules/FindPkgConfig.cmake:607 (message):
       >   A required package was not found
       > Call Stack (most recent call first):
       >   /nix/store/l341npqn3j8pwqq1nqgj5cqhnfbzhphd-cmake-3.25.1/share/cmake-3.25/Modules/FindPkgConfig.cmake:829 (_pkg_check_modules_internal)
       >   CMakeLists.txt:49 (pkg_check_modules)
       >
       > 
       > -- Configuring incomplete, errors occurred!
       > See also "/build/source/build/CMakeFiles/CMakeOutput.log".
       For full logs, run 'nix log /nix/store/15qln57gip1vlnakkiyg1pdpavvz1n7j-shotcut-22.12.21.drv'.
```

Reply:
sorry, I meant the CFLAGS line and not fftw. Github point-and-click fooled me.